Coldwave/Coldwave.entitlements

```diff
@@ -2,9 +2,13 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>com.apple.security.app-sandbox</key>
-    <true/>
-    <key>com.apple.security.files.user-selected.read-only</key>
-    <true/>
+	<key>com.apple.security.app-sandbox</key>
```

Comment:
Some of this is needed in order to make the "remember folder" work (since there is no user action.
I think there is a way to make the application ask for access to Documents, Downloads and maybe Music folder instead of forcing them here, but I am not sure of how to do that.